torchinductor/config.py

```diff
@@ -35,7 +35,7 @@
 realize_bytes_threshold = 2000
 
 # fallback to eager for random/dropout, this is slow but useful for debugging
-fallback_random = False
+fallback_random = True
```

Comment:
I suppose default this to False has a performance penalty? Have you done any measurement?

Reply:
We should leave this off by default

Reply:
accident